### ISSUE TITLE
initStyle of column doesn't work properly - VUE 3

### ISSUE THREAD

**Issue by ganevaleks** [cscan/vue3-excel-editor#39]
![Screenshot 2023-12-04 at 18 16 41](https://github.com/cscan/vue3-excel-editor/assets/15260855/b067b7cc-a7b3-4df3-bbdb-02646f7945e3)

My code 

`function cellStyle(content)
 {
    if(content !== null && content === "BREAK45")  {
        objectColor.color = "red";
        return content;
    } else {
        objectColor.color = "black";
        return content;
    }
}`

All columns of row 5 that have the string "BREAK45" should be colored red, but they're not.  

Any suggestions?

**Comment by cscan** [comment]
you should return the cell style in object, e.g. {color：red ....Sent from my phone-------- Original message --------From: ganevaleks ***@***.***>Date: 2023年12月5日週二 00:23To: cscan/vue3-excel-editor ***@***.***>Cc: Subscribed ***@***.***>Subject: [cscan/vue3-excel-editor] initStyle of column doesn't work properly - VUE 3 (Issue #39)
Screenshot.2023-12-04.at.18.16.41.png (view on web)
My code
function cellStyle(content) { if(content !== null && content === "BREAK45")  { objectColor.color = "red"; return content; } else { objectColor.color = "black"; return content; } }
All columns of row 5 that have the string "BREAK45" should be colored red, but they're not.
Any suggestions?

**Comment by ganevaleks** [comment]
yes, that's the way i'm returning it. But first elelement is not colored red and first two of row 6 are colored, but the don't have to be. 

![Screenshot 2023-12-05 at 9 23 51](https://github.com/cscan/vue3-excel-editor/assets/15260855/b85540ac-8468-4fd8-bfec-f44e741d3f64)

**Comment by cscan** [comment]
cellStyle has other arguments for you to identify which row is it.
